types/react-ranger/index.d.ts

```diff
@@ -0,0 +1,62 @@
+// Type definitions for react-ranger 2.0
+// Project: https://github.com/tannerlinsley/react-ranger#readme
+// Definitions by: Matt Polichette <https://github.com/mpolichette>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Key, CSSProperties, MouseEvent, TouchEvent } from 'react';
+
+export interface RangerOptions {
+    values: number[];
+    min: number;
+    max: number;
+    stepSize: number;
```

Comment:
I'd check with @tannerlinsley, the docs say that this is required, but it's clear from the examples that it's sometimes optional.

Reply:
I think it is required.  From the code, I don't see anything like defaults being provided, I'd rather be strict on it.  [Step size is definitely required](https://github.com/tannerlinsley/react-ranger/blob/master/src/index.js#L79-L83)

Reply:
Steps are required, stepSize is only required if steps aren't defined.  See: https://github.com/tannerlinsley/react-ranger/blob/master/examples/custom-steps/src/index.js#L9-L15

At least that's my reading of it.